docker-compose-release.yml

```diff
@@ -2,33 +2,15 @@ version: "3.6"
 
 services:
   node:
-    build:
-      context: ./
-      dockerfile: ./crates/standalone/Dockerfile
     volumes:
-      - ./crates/standalone:/usr/src/app/crates/standalone
-      - ./crates/core:/usr/src/app/crates/core
-      - ./crates/client-api:/usr/src/app/crates/client-api
-      - ./crates/lib:/usr/src/app/crates/lib
-      - ./crates/sats:/usr/src/app/crates/sats
-      - ./crates/bindings:/usr/src/app/crates/bindings
-      - ./crates/bindings-macro:/usr/src/app/crates/bindings-macro
-      - ./crates/bindings-sys:/usr/src/app/crates/bindings-sys
-      - ./crates/vm:/usr/src/app/crates/vm
-      - ./crates/metrics:/usr/src/app/crates/metrics
+      - ./crates:/usr/src/app/crates
       - key_files:/etc/spacetimedb
       - /stdb
     ports:
-      - "3000:80"
-    privileged: true
+      - "3000:3000"
```

Comment:
It's clear this file hasn't been used in a *long* time because the port was still set to 80. I would find this file useful now that I know it's been updated so I'd like to keep it.

Reply:
Can you leave a comment about what this file does, how it differs from the other dockerfile, and why it's useful?